### PR TITLE
Fix MediaCo Portal test

### PR DESCRIPTION
**Pull request by vrseraphin**
Fix attachment logic so test doesn't fail

### CODE REVIEW

File: src/bridge/BridgeBase/index.ts
Comment (niallriddell) [inline src/bridge/BridgeBase/index.ts:1]: Can yo uexplain why these changes are needed in BridgeBase?  Is this to do with the reference component issue you mentioned?

**Review by niallriddell — COMMENTED** [review]
I have one comment.

File: src/bridge/BridgeBase/index.ts
Comment (vrseraphin) [reply src/bridge/BridgeBase/index.ts:1]: Running tests without that change displays a black screen with error stacks.  See:
Screenshot 2024-02-28 at 12 40 12 PM

Comment (vrseraphin) [reply src/bridge/BridgeBase/index.ts:1]: Running without that change results in the following error screen:
Screenshot 2024-03-05 at 11 33 01 AM

Comment (niallriddell) [reply src/bridge/BridgeBase/index.ts:1]: Yes.  I'm wondering if the cause of the error is known?  Even if this is a solution?  I think this is because it doe not handle references properly.  I'm going to approve but I will add a story to explore why we get this error.

**Review by niallriddell — APPROVED** [review]
